modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/stretchr/testify/suite"
 
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
super small nit: could you group this import in the middle, [see reasoning](https://github.com/cosmos/ibc-go/pull/337#discussion_r694058339)

Reply:
No problemo!